modules/backend/assets/less/layout/fancylayout.less

```diff
@@ -71,6 +73,24 @@ body.breadcrumb-fancy .control-breadcrumb,
         }
     }
 
+    .fancy-layout {
```

Comment:
@ayumihamsaki Please remove this block and the one below, as this is not the correct location for these changes. You should instead insert your changes in line 109 below, and maybe use something like the `darken()` LESS function to apply a slight change to the colour but still use the original LESS variable.

Reply:
@bennothommo I don't think you're right with moving it to line 109 because line 94 has a `comma` not a `dot`, e.g.

```css
.control-tabs, &.control-tabs {
```

not

```css
.control-tabs. &.control-tabs {
```

Reply:
@ayumihamsaki It is correct. LESS will render that two separate lines:

```
.fancy-layout .control-tabs
.fancy-layout.control-tabs
```